src/helpers/queryHelpers.ts

```diff
@@ -60,10 +60,15 @@ export const useServiceQuery = <TItem, TColumnKey extends string, TSortableColum
   columnOrderMap?: Record<TSortableColumnKey, string>;
   tableState: TableState<TItem, TColumnKey, TSortableColumnKey>;
   setRefreshTime?: (date: Date) => void;
-}) =>
-  useQuery<ServiceQueryResult<TItem>>({
+}) => {
+  const pollInterval = process.env.REACT_APP_POLL_INTERVAL
+    ? Number.parseInt(process.env.REACT_APP_POLL_INTERVAL, 10)
+    : undefined;
```

Comment:
optional: 
some more subtle js behavior this also returns undefined... this could be simplified and we can leverage the same property name 
```
const refetchInterval = process.env.REACT_APP_POLL_INTERVAL && Number.parseInt(process.env.REACT_APP_POLL_INTERVAL, 10);

...
refetchOnWindowFocus: !helpers.DEV_MODE,
refetchInterval,
...
```